app/views/results.nunjucks

```diff
@@ -7,7 +7,7 @@
 {% endblock %}
 
 {% block content %}
-<a href="{{ SITE_ROOT }}/find-help?context={{ context }}" class="link-back">Back to find a GP</a>
+<a href="{{ SITE_ROOT }}?context={{ context }}" class="link-back">Back to find your GP</a>
```

Comment:
At the content review yesterday it was discussed that there is a need to show from where the user is coming from, so the context is still used to display the back to a specific condition/symptom a la pharmacy finder